packages/app/src/widgets/borrow-flow/components/collateral-step-modal/collateral-step-modal.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react'
+import { SupportedToken } from '@/shared/stellar-constants/tokens'
+import { ModalLayout } from '../modal-layout'
+import { APR, MINIMUM_HEALTH_VALUE, coinInfoByType } from '../../constants'
+import { formatUsd } from '../../utils'
```

Comment:
The name
```suggestion
import { formatUsd } from '../../formatters'
```
makes more sence than impersonal `utils`. What do you think?

Reply:
Renamed